src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+function strNum(str) {
+  const result = str
+    .slice(1)
+    .split(',')
+    .reduce((a, b) => a + b);
```

Comment:
The `strNum` function attempts to convert a string to a number by slicing, splitting, and reducing. However, it does not handle the conversion correctly. Ensure that each part of the string is converted to a number before summing them up. Consider using `parseFloat` or `parseInt` on each element after splitting.